nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/pom.xml

```diff
@@ -32,6 +32,16 @@
             <artifactId>nifi-utils</artifactId>
             <version>1.10.0-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-lookup-service-api</artifactId>
+            <version>1.10.0-SNAPSHOT</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-lookup-services</artifactId>
+            <version>1.10.0-SNAPSHOT</version>
+        </dependency>
```

Comment:
We don't want to bundle `nifi-lookup-services` into multiple NARs, it should only be bundled in the standard services NAR, and other NARs use those services through the API.

I think the reason this was included was because that is where the abstract lookup service lives, so I would say there are two approaches...

1) In `nifi-lookup-services-bundle`, create another module like  `nifi-lookup-service-utils` and put the abstract class there and update these dependencies to the utils module so that we are separating it from implementations.

2) Use, or create a new module, under `nifi-nar-bundles/nifi-extension-utils`, in most cases abstract classes reused by processors have ended up in `nifi-processor-utils`, so maybe that could be used, or maybe a new service utils module.

Reply:
@bbende Thanks, that makes sense. I created a `nifi-service-utils` under `nifi-nar-bundles/nifi-extension-utils` and moved the abstract lookup there.